test/local-peers.js

```diff
@@ -307,6 +324,130 @@ test('connected peer has protomux instance', async () => {
   assert(Protomux.isProtomux(peer.protomux))
 })
 
+test('Device info with ack results in acks sent', async () => {
+  const r1 = new LocalPeers()
+  const r2 = new LocalPeers()
+
+  /** @type {import('../src/generated/rpc.js').DeviceInfo} */
+  const expectedDeviceInfo = {
+    name: 'mapeo',
+    deviceType: 'mobile',
+    features: [DeviceInfo_RPCFeatures.ack],
+  }
```

Comment:
maybe a stylistic thing but was expecting the test to make some assertion on this because of the `expect` prefix in the name. found it a bit confusing that it's only used as an input (albeit the one that affects the result of the test)

Reply:
renamed to `validDeviceInfo`